operators/nlp-server/0.8.0-rc1/manifests/nlp-server.clusterserviceversion.yaml

```diff
@@ -77,10 +77,10 @@ metadata:
       ]
     capabilities: Basic Install
     categories: AI/Machine Learning
-    containerImage: registry.johnsnowlabs.com/nlp-server/nlp-server-operator:0.8.0-rc1
+    containerImage: registry.johnsnowlabs.com/nlp-server/nlp-server-operator:0.8.6
     operators.operatorframework.io/builder: operator-sdk-v1.23.0
     operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
-  name: nlp-server.v0.8.0-rc1
+  name: nlp-server.v0.8.6
```

Comment:
@ravienggtoo if you are changing version here, please rename operator version directory to this value.

Reply:
Hi @mvalarh , Thanks for the suggestion . 

I have renamed the directory and kindly let me know if it fine.

Reply:
thanks @ravienggtoo